File: cSharp/problem-1011.cs
using System; 

// https://www.beecrowd.com.br/judge/en/problems/view/1011
class URI {

    static void Main(string[] args) { 

        double pi = 3.14159;
        int radius = int.Parse(Console.ReadLine());
        double sphereVolume = (4.0 / 3) * pi * Math.Pow(radius, 3);
        Console.WriteLine($"VOLUME = {sphereVolume.ToString("0.000")}");
    }

}

File: c/problem-1011.c
#include <stdio.h>
#include <math.h>

int main() {

    int radius;
    scanf("%d", &radius);
    double PI = 3.14159, sphere_volume = (4 / 3.0) * PI * pow(radius, 3);

    printf("VOLUME = %.3lf\n", sphere_volume);
    return 0;
}
